src/components/CommentsByReviewId.jsx

```diff
@@ -7,34 +7,44 @@ import * as React from "react";
 import Button from "@mui/material/Button"
 import DeleteOutlineIcon from '@mui/icons-material/DeleteOutline';
 import ThumbDownOffAltIcon from '@mui/icons-material/ThumbDownOffAlt';
-
-
+import DeleteComment from "./DeleteComment";
+import { toast } from "react-toastify"
+import { DeleteCommentOnReview } from '../api';
 
 function CommentsByReviewId({isLightTheme, user, id}) {
   const [newCommentsById, setNewCommentsById] = useState(null);
   const [loading, setLoading] = useState(false);
   const { review_id } = useParams();
-  
+  const [deletedComment, setDeletedComment] = useState(false)
   function themeoptions(lightoption, darkoption) {
     
     if (isLightTheme) {
       return lightoption
     } else {return darkoption} 
   }
 
+  function handleDelete(comment_id) {
+   
+    DeleteCommentOnReview(comment_id).then(() => {
+      setDeletedComment(true)
+        toast.success(`Comment removed!`,  { position: toast.POSITION.TOP_RIGHT })
+    }).catch(() => {
+        toast.error(`Cannot remove comment`,  { position: toast.POSITION.TOP_RIGHT });
+    })
+    }
   
   useEffect(() => {
     setLoading(true)
     fetchCommentsByReviewId(review_id).then((  comments  ) => {
-      setNewCommentsById(comments);
+      
+      const sortedComments = comments.comments.sort((a,b)=>b.comment_id>a.comment_id ? 1 : -1) 
+     
+      setNewCommentsById(sortedComments);
       setLoading(false)
     });
-    }, []); 
+    }, [deletedComment]); 
```

Comment:
The comments shouldnt be refetched from the api after deleting. 
This is slower, but also if other people are deleting or adding, you'll get a weird result where more or less items appear compared to when you deleted. 
It will look like you've accidentally deleted multiple things, or added some somehow.